src/themes/components/carousel.ts

```diff
@@ -96,6 +96,64 @@ const numbersPaginationButton = {
   },
 };
 
+const dotsPaginationButton = {
```

Comment:
I think you should increase the click area a bit. It is very hard to hit the dot because it's so small. Easy to missclick. Maybe you don't even need it. From refinment I only understood it as an indicator and not actually clickable?